colorclade/_biopython_draw_colorclade_tree.py

```diff
@@ -98,15 +109,17 @@ def biopython_draw_colorclade_tree(
             biopy_tree,
             axes=ax,
             label_func=lambda node: (
-                node.name if label_tips and node.is_terminal() else ""
+                node.name if label_tips(node) and node.is_terminal() else ""
             ),
+            label_colors=color_labels,
             do_show=False,
         )
 
     if drop_overlapping_labels:
         matplotlib_drop_overlapping_labels(ax)
 
     # Remove axes borders except for bottom, and remove y-axis tick/labels
+    ax: plt.Axes
```

Comment:
I’d move this to right after all the ax processing and add a comment like “ax is now guaranteed to be…”